### PR TITLE
added footer

### PR DESCRIPTION
**Pull request by BhuwanChandra**
feat: Add a footer

Added footer in the component Footer.js

footer

Resolves: #34

### CODE REVIEW

**Comment by BhuwanChandra** [comment]
Ok, I will change the font size

**Comment by BhuwanChandra** [comment]
@ankitjena I have reduced the font size.

footer

File: components/Footer.js [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+    style={{
+      paddingTop: "15px",
+      paddingBottom: "15px",
+      position: "absolute",
+      bottom: 0,
+      width: "100%"
+    }}
+    className="bg-primary"
+  >
+    <footer className="">
```
Comment (rkpattnaik780) [inline components/Footer.js]: Remove the `className` attribute as no class is being used.

File: components/Footer.js [hunk 2]
```diff
@@ -1,12 +1,58 @@
...
+      <div className="text-center">
+        <div style={{fontSize: "16px"}}>
+          Made with <span style={{ color: "#ad1010" }}>❤</span> for
```
Comment (rkpattnaik780) [inline components/Footer.js]: Better if we use the HTML symbol entities. You can check this one https://www.w3schools.com/charsets/tryit.asp?deci=9829&ent=hearts.

**Comment by BhuwanChandra** [comment]
@coderwassananmol , @rkpattnaik780 changes Done!

footer

**Pull request merged**